packages/remix/src/ssr/types.ts

```diff
@@ -49,6 +49,6 @@ export type LoaderFunctionArgsWithAuth<Options extends RootAuthLoaderOptions = a
 
 export type RequestWithAuth<Options extends RootAuthLoaderOptions = any> = LoaderFunctionArgs['request'] & {
   auth: Omit<AuthObject, 'session' | 'user' | 'organization'>;
-} & (Options extends { loadSession: true } ? { session: Session | null } : {}) &
-  (Options extends { loadUser: true } ? { user: User | null } : {}) &
-  (Options extends { loadOrganization: true } ? { organization: Organization | null } : {});
+} & (Options extends { loadSession: true } ? { session: Session | null } : object) &
+  (Options extends { loadUser: true } ? { user: User | null } : object) &
+  (Options extends { loadOrganization: true } ? { organization: Organization | null } : object);
```

Comment:
I don't think this change affects type inference at all, but I want to ask just to make sure - is there a possibility that this change affects in any way the type for consumers?

Reply:
See the previous comment. `object` should be the same behavior as `{}`